<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="browseButton">
       <property name="text">
        <string>Open ...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pathLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="imagePreviewLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>No File Selected</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>browseButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>openSelectImageFileDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>51</x>
     <y>-6</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>selectedImageFileChanged(QPixmap)</signal>
  <slot>openSelectImageFileDialog()</slot>
 </slots>
</ui>
